sample/src/androidTest/java/uk/gov/hmrc/components/sample/TabBarTest.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.hmrc.components.sample
+
+import org.junit.Test
+import uk.gov.hmrc.components.sample.rules.FontScale
+import uk.gov.hmrc.components.sample.rules.NightMode
+
+class TabBarTest : BaseActivityTest() {
+
+    @Test
+    fun screenshots() {
```

Comment:
Should these functions start with `override`? The definitions in `BaseActivityTest` are marked as open

Reply:
The tab bar test is slightly different to the rest so doesn't override any functions